maintainers/team-list.nix

```diff
@@ -699,8 +699,11 @@ with lib.maintainers; {
   rust = {
     members = [
       andir
```

Comment:
Don't think I knew that this team existed, must have been generated from codeowners.

Please remove `andir`, they removed themselves treewide in 31e5b8dc2198eacec3e009e21fc8226486aa674e, I think they're only listed here because they used to be a codeowner.